dev/scripts/release.py

```diff
@@ -0,0 +1,121 @@
+import subprocess
+from subprocess import PIPE
+import itertools
+import shutil
+from pathlib import Path
+import sys
+
+
+EXCLUDE_DIRS = [
+    '.git',
+    '.github',
+    '.idea',
+    '.node-cache',
+    '.vscode',
+    'node_modules',
+    'release',
+]
```

Comment:
I think we should add `cache` here. Currently cache folder is copied to the generated zips which we don't want

Reply:
`cache` contains an important `.htaccess` file, so I don't think it should be excluded